orchagent/portsorch.cpp

```diff
@@ -2351,6 +2426,9 @@ bool PortsOrch::initPort(const string &alias, const string &role, const int inde
                 /* Create associated Gearbox lane mapping */
                 initGearboxPort(p);
 
+                /* Initialize port capabilities */
+                initPortCapLinkTraining(p);
```

Comment:
This function loops through all the ports and get capability. This would increase the port initialization time which is a bottleneck in fastboot scenarios. If the user has no LT configs this operation is not needed.
Instead init the capability only when the user configures LT and capability not in cache.

Reply:
Noted, and it's done